### ISSUE TITLE
The score is keeps zero

### ISSUE THREAD

**Issue by hyyh28** [dnddnjs/feudal-montezuma#7]
I have make it train more than 700 episodes but the score still keeps 0. In the author's paper, the montezuma_revenge has got a great improving after 200.

**Comment by jdubkim** [comment]
Hi, it's really good to hear that you tried to use our code! However, unfortunately, we're still working on the code. Even though we implemented the structure of FuN, we still need to implement dilated LSTM and A2C. Since we're still working on this project really hard, please keep up with this repository :) We are aiming to finish this within 2 weeks.

**Comment by zhkmxx9302013** [comment]
> Hi, it's really good to hear that you tried to use our code! However, unfortunately, we're still working on the code. Even though we implemented the structure of FuN, we still need to implement dilated LSTM and A2C. Since we're still working on this project really hard, please keep up with this repository :) We are aiming to finish this within 2 weeks.

Thanks for working on the code. I want to know do you have any reference when you building the FuN or how long could you update this repository, thanks :)

**Comment by zhkmxx9302013** [comment]
> I have make it train more than 700 episodes but the score still keeps 0. In the author's paper, the montezuma_revenge has got a great improving after 200.

I find that you can try to increase the RMSprop learning rate first.

**Comment by jdubkim** [comment]
The only reference we had for building the FuN was the paper uploaded on Arxiv. That's we're still working on this project. Currently, we're working in 'dLSTM' branch 'lstm_a2c' folder, but the network is not getting trained well. We're still working on this project, and since the deadline for our project is Feb 23rd, we can assure you that we're gonna update this repo frequently until then (or more if we decide to continue this project.) Thanks for your interests :)
